static/app/views/settings/organizationApiKeys/organizationApiKeysList.tsx

```diff
@@ -101,21 +101,25 @@ function OrganizationApiKeysList({
               </TextCopyInput>
 
               <Cell>
-                <LinkWithConfirmation
-                  aria-label={t('Remove API Key')}
-                  className="btn btn-default btn-sm"
+                <Confirm
                   onConfirm={() => onRemove(id)}
                   message={t('Are you sure you want to remove this API key?')}
-                  title={t('Remove API Key?')}
                 >
-                  <IconDelete
-                    size="xs"
-                    css={css`
-                      position: relative;
-                      top: 2px;
-                    `}
-                  />
-                </LinkWithConfirmation>
+                  <Anchor
+                    href="#"
```

Comment:
wondering why this is a link in the first place instead of a button with the `IconDelete` ? It doesn’t go anywhere - it performs a delete action :/

Reply:
let me try and find it in the UI so I can see what it does and try replacing it.